README.md

```diff
@@ -134,68 +115,24 @@ Another option for looking up registries is to store an element's originating re
 
-In the other direction, we want to be able to call `new MyElement()`, which means we need to be able to locate the correct tag name from a constructor as well.
-
-The way this is done is by limiting constructors by default to only looking up registrations from the global registry. If the constructor is not defined in the global registry, it will throw. In order to get a constructor that creates a scoped definition, customElementRegistry.define() returns a new constructor:
-
-```js
-import {OtherElement} from './my-element.js';
-
-const registry = new CustomElementRegistry();
-
-// define() returns a new class:
-const LocalOtherElement = registry.define('other-element-2', OtherElement);
-const el = new LocalOtherElement();
-el.tagName === 'other-element-2';
-
-// The same class is available from registry.get():
-const O = registry.get('other-element-2')
-const el2 = new O();
-el2.tagName === 'other-element-2';
-```
-
-The constructor returned by `define()` is from a trivial subclass of the registered class.
-
-### Sugar for bulk registrations
-
-To make importing and registering multiple definitions easier, the CustomElementRegistry constructor can take an object containing multiple definitions:
+As a result, it must limit constructors by default to only looking up registrations from the global registry. If the constructor is not defined in the global registry, it will throw.
 
-```js
-import {ElementOne} from './element-one.js';
-import {ElementTwo} from './element-two.js';
-
-const registry = new CustomElementRegistry({
-  definitions: {
-    'element-one': ElementOne,
-    'element-two': ElementTwo,
-  }
-});
-
-export class MyElement extends HTMLElement {
-  constructor() {
-    this.attachShadow({mode: 'open', registry});
-  }
-}
-```
+This poses a limitation for authors trying to use the constructor to create new elements associated to scoped registries but not registered as global. More flexibility can be analyzed post MVP, for now, a user-land abstraction can help by keeping track of the constructor and its respective registry.
 
 ## API
 
 ### CustomElementRegistry
 
 #### Constructor
 
-The CustomElementRegistry constructor creates a new instance of CustomElementRegistry, independent of the instance available at `window.customElements`.
+The `CustomElementRegistry` constructor creates a new instance of CustomElementRegistry, independent of the instance available at `window.customElements`.
 
 ##### Syntax
 
 ```js
-const registry = new CustomElementRegistry({parent, definitions});
+const registry = new CustomElementRegistry();
 ```
 
-##### Parameters
```

Comment:
As discussed in https://github.com/justinfagnani/webcomponents/pull/1#discussion_r407597061, we can explore the ability to define a batch of components and their names via `CustomElementRegistry.prototype.define` instead of using a options bag in the constructor. /cc @leobalter